docs/reference/query-languages/esql/esql-search-tutorial.md

```diff
@@ -459,16 +399,244 @@ FROM cooking_blog METADATA _score
+
+ES|QL supports semantic search when your mappings include fields of the [`semantic_text`](/reference/elasticsearch/mapping-reference/semantic-text.md) type. This example mapping update adds a new field called `semantic_description` with the type `semantic_text`:
+
+```console
+PUT /cooking_blog/_mapping
+{
+  "properties": {
+    "semantic_description": {
+      "type": "semantic_text"
+    }
+  }
+}
+```
+
+Next, index a document with content into the new field:
+
+```console
+POST /cooking_blog/_doc
+{
+  "title": "Mediterranean Quinoa Bowl",
+  "semantic_description": "A protein-rich bowl with quinoa, chickpeas, fresh vegetables, and herbs. This nutritious Mediterranean-inspired dish is easy to prepare and perfect for a quick, healthy dinner.",
+  "author": "Jamie Oliver",
+  "date": "2023-06-01",
+  "category": "Main Course",
+  "tags": ["vegetarian", "healthy", "mediterranean", "quinoa"],
+  "rating": 4.7
+}
+```
+
+### Perform semantic search
+
+Once the document has been processed by the underlying model running on the inference endpoint, you can perform semantic searches. Use the [`MATCH` function](/reference/query-languages/esql/functions-operators/search-functions.md#esql-match) (or the [`:` operator](/reference/query-languages/esql/functions-operators/operators.md#esql-match-operator) shorthand) on `semantic_text` fields to perform semantic queries:
+
+```esql
+FROM cooking_blog METADATA _score
+| WHERE semantic_description:"I need plant-based meals that aren't difficult to make"
+| SORT _score DESC
+| LIMIT 5
+```
+
+When you use `MATCH` or `:` on a `semantic_text` field, {{esql}} automatically performs a semantic search rather than a lexical search. This means the query finds documents based on semantic similarity, not just keyword matching.
+
+:::{tip}
+If you'd like to test out the semantic search workflow against a large dataset, follow the [semantic-search-tutorial](docs-content://solutions/search/semantic-search/semantic-search-semantic-text.md).
+:::
+
+### Perform hybrid search
+
+Hybrid search combines different search strategies (like lexical and semantic search) to leverage the strengths of each approach. {{esql}} provides two ways to implement hybrid search:
+
+::::{tab-set}
+
+:::{tab-item} `FORK` and `FUSE`
+```{applies_to}
+stack: preview 9.2
+serverless: preview
+```
+
+Use [`FORK`](/reference/query-languages/esql/commands/fork.md) and [`FUSE`](/reference/query-languages/esql/commands/fuse.md) to execute different search strategies in parallel, then merge and score the combined results.
+
+```esql
+FROM cooking_blog METADATA _id, _index, _score
+| FORK (
+    WHERE title:"vegetarian curry" <1>
+    | SORT _score DESC
+) (
+    WHERE semantic_description:"easy vegetarian curry recipes" <2>
+    | SORT _score DESC
```

Comment:
let's add a `| LIMIT 5` here since we plan to remove the implicit limit for fork - and then this query might actually break.
same as with the other branch - let's make sure they both have a LIMIT